recipes/libdeflate/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.3" %}
+{% set sha256 = "09a23f5a6bf74789605a452364fae2156627a0a62a3de2a82ceb362982a87f0b" %}
+
+package:
+  name: libdeflate
+  version: {{ version }}
+
+source:
+  url: https://github.com/ebiggers/libdeflate/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Build-dylib-on-MacOS.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('libdeflate', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libdeflate.a            # [unix]
+    - test -f ${PREFIX}/lib/libdeflate${SHLIB_EXT}  # [unix]
```

Comment:
Sorry to be too late here: Would `- conda inspect linkages {{ name }}  # [not win]` improve testing here.

Reply:
I'm not sure, it only links against libc and vdso, so there's no much to see.